doc/refmodel.rst

```diff
@@ -47,19 +47,18 @@ Methods
 * ``solve(m::Model; suppress_warnings=false, relaxation=false)`` - solves the model using the selected solver (or a default for the problem class), and takes two optional arguments that are disabled by default. Setting ``suppress_warnings`` to ``true`` will suppress all JuMP-specific output (e.g. warnings about infeasibility and lack of dual information) but will not suppress solver output (which should be done by passing options to the solver). Setting ``relaxation=true`` solves the standard continuous relaxation for the model: that is, integrality is dropped, special ordered set constraints are not enforced, and semi-continuous and semi-integer variables with bounds ``[l,u]`` are replaced with bounds ``[min(l,0),max(u,0)]``.
-* ``buildInternalModel(m::Model)`` - builds the model in memory at the MathProgBase level without optimizing.
-* ``setSolver(m::Model,s::AbstractMathProgSolver)`` - changes the solver which will be used for the next call to ``solve()``, discarding the current internal model if present.
-* ``getVar(m::Model,name::Symbol)`` - returns the variable or group of variables of the given name which were added to the model with ``@defVar``. Throws an error if multiple variables were created with the same name.
+* ``JuMP.build(m::Model)`` - builds the model in memory at the MathProgBase level without optimizing.
+* ``setsolver(m::Model,s::AbstractMathProgSolver)`` - changes the solver which will be used for the next call to ``solve()``, discarding the current internal model if present.
+* ``getvariable(m::Model,name::Symbol)`` - returns the variable or group of variables of the given name which were added to the model with ``@defVar``. Throws an error if multiple variables were created with the same name.
 
 **Objective**
 
-* ``getObjective(m::Model)`` - returns the objective function as a ``QuadExpr``.
-* ``setObjective(m::Model, sense::Symbol, a::AffExpr)``, ``setObjective(m::Model, sense::Symbol, q::QuadExpr)`` - sets the objective function to ``a`` and ``q`` respectively, with given objective sense, which must be either ``:Min`` or ``:Max``.
-* ``getObjectiveSense(m::Model)`` - returns objective sense, either ``:Min`` or ``:Max``.
-* ``setObjectiveSense(m::Model, newSense::Symbol)`` - sets the objective sense (``newSense`` is either ``:Min`` or ``:Max``).
-* ``getObjectiveValue(m::Model)`` - returns objective value after a call to ``solve``.
+* ``getobjective(m::Model)`` - returns the objective function as a ``QuadExpr``.
+* ``getobjectivesense(m::Model)`` - returns objective sense, either ``:Min`` or ``:Max``.
+* ``setobjectivesense(m::Model, newSense::Symbol)`` - sets the objective sense (``newSense`` is either ``:Min`` or ``:Max``).
+* ``getobjectivevalue(m::Model)`` - returns objective value after a call to ``solve``.
```

Comment:
@mlubin should this be ``objectivevalue`` instead, given our naming convention? (you can't "set" the objective value)